renderdoc/common/globalconfig.h

```diff
@@ -104,6 +104,15 @@
 #define RDOC_POSIX OPTION_ON
 #define RDOC_GGP OPTION_ON
 
+#elif defined(RENDERDOC_PLATFORM_SWITCH)
+
+#define RDOC_WIN32 OPTION_OFF
+#define RDOC_ANDROID OPTION_OFF
+#define RDOC_LINUX OPTION_OFF
+#define RDOC_APPLE OPTION_OFF
+#define RDOC_POSIX OPTION_ON
+#define RDOC_GGP OPTION_OFF
```

Comment:
We should add a new `RDOC_SWITCH` option here and use that in preference to `RENDERDOC_PLATFORM_SWITCH` wherever possible. It should only be needed either to enable that option here, or in `apidefs.h` which is a public header and so can't include this.

The other uses in common.h, gl_common.h and so on should then use `RDOC_SWITCH`.

Reply:
I understand.  Will do.